packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -0,0 +1,93 @@
+/** @jsx withSlots */
+import {
+  applyTokenLayers,
+  buildUseTokens,
+  compressible,
+  mergeStyles,
+  Theme,
+  withSlots,
+  useSlot,
+  useTheme,
+  UseTokens,
+  borderStyles,
+} from '@fluentui-react-native/framework';
+import { ShadowProps, ShadowTokens } from './Shadow.types';
+import { View, ViewProps } from 'react-native';
+import { applyPropsToTokens } from '@fluentui-react-native/use-tokens';
+
+const useShadowTokens = buildUseTokens<ShadowTokens>((theme: Theme) => {
+  return {
+    color: 'black',
+    backgroundColor: theme.colors.background,
+    ambient: theme.effects.shadow4.ambient,
+    key: theme.effects.shadow4.key,
+    innerShadowSlot: View,
+    depth2: {
+      ambient: theme.effects.shadow2.ambient,
+      key: theme.effects.shadow2.key,
+    },
+    depth4: {
+      ambient: theme.effects.shadow4.ambient,
+      key: theme.effects.shadow4.key,
+    },
+    depth8: {
+      ambient: theme.effects.shadow8.ambient,
+      key: theme.effects.shadow8.key,
+    },
+    depth16: {
+      ambient: theme.effects.shadow16.ambient,
+      key: theme.effects.shadow16.key,
+    },
+    depth28: {
+      ambient: theme.effects.shadow28.ambient,
+      key: theme.effects.shadow28.key,
+    },
+    depth64: {
+      ambient: theme.effects.shadow64.ambient,
+      key: theme.effects.shadow64.key,
+    },
+  };
+}, 'Shadow');
+
+export const Shadow = compressible<ShadowProps, ShadowTokens>((props: ShadowProps, useTokens: UseTokens<ShadowTokens>) => {
```

Comment:
My "I'm dumb, plz explain", overarching question:

Why does this need to be a separate component instead of just an extra set of styles that are applied to existing components? In the end the Shadow tokens are just being converted into styles that are applied to a view, why can't we just apply it to the root view of existing components?